sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -0,0 +1,111 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Example RunInference API Pipelines
+
+This module contains example pipelines that use the Beam RunInference
+API. <!---TODO: Add link to full documentation on Beam website when it's published.-->
+
+## Pre-requisites
+
+You must have `apache-beam>=2.40.0` installed in order to run these pipelines,
+because the `apache_beam.examples.inference` module was added in that release.
+```
+pip install apache-beam==2.40.0
+```
+
+### Pytorch dependencies
+The RunInference API has support for the Pytorch framework. To use Pytorch locally, first install `torch`.
+```
+pip install torch==1.11.0
+```
+
+For installation of the `torch` dependency for Dataflow pipelines, refer to these
+[instructions](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#pypi-dependencies).
+
+<!---
+TODO: Add link to full documentation on Beam website when it's published.
+
+i.e. "See the
+[documentation](https://beam.apache.org/documentation/dsls/dataframes/overview/#pre-requisites)
+for details."
+-->
+
+### Datasets and Models for RunInference
+Data related to RunInference has been staged in
+`gs://apache-beam-ml/` for use with these example pipelines. You can see this by using the [gsutil tool](https://cloud.google.com/storage/docs/gsutil#gettingstarted).
```

Comment:
(optional) maybe link to the cloud console here: https://pantheon.corp.google.com/storage/browser/apache-beam-ml

Reply:
Is that link accessible to only Google employees though? would `https://console.cloud.google.com/` be better?

Reply:
whoops, yes it would. I copied the wrong thing :)

Reply:
https://console.cloud.google.com/storage/browser/apache-beam-ml

Reply:
Done.